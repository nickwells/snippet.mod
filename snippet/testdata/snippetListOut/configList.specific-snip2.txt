    /home/ngw/go/src/github.com/nickwells/snippet.mod/snippet/testdata/testListConfig/snip2
            Note: snip2 - Note
                  snip2 - Notes
                  snip2 - Doc
                  snip2 - Docs
                  snip2 - note
                  snip2 - notes
                  snip2 - doc
                  snip2 - docs
         Imports: snip2/xxx
         Expects: snip1
         Follows: snip1
        Declares: __snip2XXX
