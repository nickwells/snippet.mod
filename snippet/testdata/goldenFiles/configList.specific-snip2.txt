in: testdata/testListConfig

    snip2/snip2.1
            Note: snip2 - Note
                  snip2 - Notes
                  snip2 - Doc
                  snip2 - Docs
                  snip2 - note
                  snip2 - notes
                  snip2 - doc
                  snip2 - docs
         Imports: snip2/xxx
         Follows: snip1
        Declares: __snip2XXX
